### ISSUE TITLE
[TnH] Can't load map for a 2nd play if returning to TnH lobby.

### ISSUE THREAD

**Issue by ViniloTraiteto** [ViniloTraiteto/Shortscope#7]
Issue is you do one run on Shortscope, go back to the TnH Lobby via scoreboard button and then you can't eat the hotdog again.

Logs said something about "Assets already being loaded" whatever that meant.

**Comment by ViniloTraiteto** [comment]
Ok, removed some unused stuff from the project/scene and it seems to be fixed.

Things I removed from scene:
- Sandbox logic _(even though it was disabled)_
- NavMeshCleaner prefab
- Unused CSG objects

I also did the export on a fresh project that didn't have:
- RealtimeCSG
- That one tool I use to change the editor's FoV
- Basically anything that isnt MeatKit, Atlas or Bakery


TODO: Try the export from the original project, only removing the Sandbox logic. Would be sad if I have to do that since I liked keeping everything in one scene.

OH OH WAIT I CAN HAVE THE MESHES AND LIGHT BAKE IN A PREFAB AND USE A DIFFERENT SCENE FOR THE LOGIC, THAT WAY I DONT HAVE TO REPEAT FIXES ON DIFFERENT SCENES BUT I ALSO DONT HAVE ISSUES DEALING WITH MULTIPLE GAME LOGICS ON THE SAME SCENE!.
